src/preProcessPattern.js

```diff
@@ -13,7 +12,7 @@ export default function preProcessPattern(globalRef, pattern) {
 
     pattern = typeof pattern === 'string' ? {
         from: pattern
-    } : _.cloneDeep(pattern);
+    } : Object.assign({}, pattern);
```

Comment:
That's not going to work. You will need to add a lib that actually does a deep object clone.

Reply:
@d3viant0ne This line doesn't have sense with merge deep